nighthawk/source/client/BUILD

```diff
@@ -0,0 +1,48 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:nighthawk.bzl",
+    "nighthawk_cc_platform_dep"
+)
+
+load(
+    "@envoy//bazel:envoy_build_system.bzl",
+    "envoy_cc_binary",
+    "envoy_cc_library",
+    "envoy_cc_posix_library",
+    "envoy_cc_win32_library",
+)
+
+load("@envoy//api/bazel:api_build_system.bzl", "api_proto_library_internal")
+load("@envoy//bazel:repositories.bzl", "PPC_SKIP_TARGETS")
+
+api_proto_library_internal(
+    name = "benchmark_options",
+    srcs = ["options.proto"],
+    visibility = ["//visibility:public"],
+)
+
+envoy_cc_library(
+    name = "nighthawk_client_lib",
+    srcs = [
+        "client.cc",
+	    "options_impl.cc",
```

Comment:
Nit: weird indent here and below; should we run `buildifer` as part of a `check_format` phase on this repo? We can probably take the main Envoy repos' `check_format` and strip it down to just the basics, e.g. `clang-format` and `buildifier`.

Reply:
+1 for automatically running formatting tools.

Reply:
(Just applied `buildifier` to all BUILD files)